hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/exceptions/OMException.java

```diff
@@ -223,6 +223,6 @@ public String toString() {
 
     INVALID_VOLUME_NAME,
 
-    REPLAY // When ratis logs are replayed.
+    REPLAY
```

Comment:
Any reason for keeping this around?

Reply:
Initially, the reason was  because proto.lock file, if any breaking changes, it will fail to compile. I have removed this, as anyway HA is part of this release, and we can remove this field.

cc @avijayanhwx

Reply:
The general guideline is breaking API changes are allowed pre-GA, but not after that.